docs/overview.rst

```diff
@@ -451,3 +451,71 @@ infrastructure, Linux and SSH. A good place to start is Heroku (https://www.hero
 We reference Heroku in our tutorials, but Agile Toolkit can work with any cloud
 hosting that runs PHP apps.
 
+
+Online-Demo showing many functions
+==================================
+
+There's a demo available of atk4/ui which shows many of the modules and functions available
+in atk4.
+You can watch & use the source code of each example to find best practice examples and to see
+how to use atk4 in certain application cases.
+
+You can find the demo here:
+https://agile-ui.readthedocs.io/en/latest/index.html
+
+
+Demo contained in the atk4 repository
+=====================================
+When you download and install atk4 you will find a subdirectory called "demos" in the atk4
+repository which also could be locally executed.
+This folder is basically intended to run under the UI package only.
+That means the demo may not work properly, if you copy the "demos" directory out of this location
+(to run it from a different location) or if the "vendors" directory isn't accessible through the
```

Comment:
I thought the above fix addresses that? 

Could you rephrase this text in the form of step-by-step instruction on how to actually run demo elsewhere?